### DISCUSSION TITLE
TｉｋTｏｋ進軍 VR

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#406] [General]
RT 在歐美 TikTok 確實是良好的社群媒體
但在亞洲大部分都是，抖音一響 父母白養😼

https://technews.tw/2022/06/12/bydance-is-going-to-heavily-invest-on-vr/

**Comment by lx3211** [comment]
其實tiktok很多玩法都滿創新的，只是跟風潮還有一些奇妙的價值觀也隨之散播...

**Comment by brian431** [comment]
實在不怎麼喜歡短影片的模式，躺著然後一直被餵影片，時間很快就過去了，然而卻未必有什麼收穫。甚至不一定能達到放鬆的效果，畢竟一個皆著一個不同類型的影片一直看下去也是挺累的。

**Comment by NcuJerryKao** [comment]
Tiktok 跟抖音不一樣喔，前者是國際板的

**Comment by BingRong1217** [comment]
抖音也有蠻多負面的新聞

**Comment by Yilin20021016** [comment]
TikTok/抖音是一個成功的社群媒體 只是上面的人有點問題

**Comment by HsuChen88** [comment]
有問題的似乎不是TikTok本身，而是使用的人有問題

**Comment by larry40305** [comment]
無論是tiktok還是抖音，上面都有許多不知道腦袋裝什麼的挑戰

**Comment by Valentinaa0214** [comment]
現在越來越多社群平台上推出短影音功能，用戶也隨之增加，這在未來勢必是個趨勢。
而短影音的類型越來越多元，能夠進軍VR感覺是很創新的結合。

**Comment by Vincent20030314** [comment]
抖音跟tiktok雖然給了我們很多樂趣，但是也帶來一些負面的風氣。

**Comment by yjpxi** [comment]
在中國，抖音的普及程度極高，但也引發了對其社會影響的擔憂。一些家長和社會觀察家認為，抖音上充斥著大量的低質量娛樂內容，可能會對年輕一代的價值觀和行為產生負面影響。
